CenterForOpenScience/ember-osf-web
# [ENG-638]Reorder contributors through dragging and dropping.

- Ticket: [ENG-638]
- Feature flag: n/a

## Purpose

Upgrade `ember-sortable` to 2.2. Use the new version for editable contributor mgmt widget. Modify the collections contributor widget to use the new version of `ember-sortable`.

## Side Effects



## QA Notes

Make sure the collections contributor widget still works.


[ENG-638]: https://openscience.atlassian.net/browse/ENG-638
